docs/user/metrics/timespan.md

```diff
@@ -67,6 +67,21 @@ assertTrue(Auth.loginTime.testHasValue())
 assertTrue(Auth.loginTime.testGetValue() > 0)
 ```
 
+### Raw API
+
+> **Note**: It's recommended to use the timer API listed above.
+
+It's possible to explicitly set the timespan value, in nanoseconds.
+This API should only be used if your library or application requires recording times in a way that can not make use of `start`/`stop`/`cancel`.
```

Comment:
nit: should we add a specific example here? e.g. storing the values of low-level os timers?

Reply:
Hm, it's currently nowhere used so I have a hard time coming up with an example

Reply:
Sync was planning on use this to report the time it took for some sync operation to complete on the Rust side. Another example could be the use of some low level API that just exposes elapsed times